pom.xml

```diff
@@ -9,6 +9,7 @@
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<log4j2.version>2.13.1</log4j2.version>
+		<lombok.version>1.18.12</lombok.version>
```

Comment:
Wanneer beide plugins toch andere versienummers gebruiken is het misschien handiger de versie niet op te geven als property. Nu wordt de property maar op 1 plek gebruikt..

Reply:
Gedaan.